rnd/autogpt_server/autogpt_server/blocks/__init__.py

```diff
@@ -20,7 +20,19 @@
 
 # Load all Block instances from the available modules
 AVAILABLE_BLOCKS = {}
-for cls in Block.__subclasses__():
+
+
+def all_subclasses(clz):
+    subclasses = clz.__subclasses__()
+    for subclass in subclasses:
+        subclasses += all_subclasses(subclass)
+    return subclasses
+
+
+for cls in all_subclasses(Block):
+    if not cls.__name__.endswith("Block"):
```

Comment:
This is quite a stealthy breaking change - all blocks must now end with Block else they will silently not be shown by the frontend.

Reply:
i.e The block `YouTubeTranscriber` is no longer available, and no error or warning appears if you have a block that is being ignored because of this.

Documentation will also need to be updated if this is unavoidable.